src/inputs/input/input.jsx

```diff
@@ -226,10 +227,11 @@ const defaultProps = {
     style: null,
     tabIndex: null,
     type: null,
+    value: '',
```

Comment:
NICE!  This will get rid of the silly "switching back and forth between controlled and uncontrolled" issue, right?

Reply:
No, I don't believe this line disrupts that behavior.

Reply:
Oh, dang, I thought it might, since it was defaulting the value to empty string (which we usually have to do manually ourselves on the consumer side)